librz/type/parser/c_cpp_parser.c

```diff
@@ -168,10 +180,12 @@ static int type_parse_string(CParserState *state, const char *code, char **error
 	for (i = 0; i < root_node_child_count; i++) {
 		TSNode child = ts_node_named_child(root_node, i);
 		// We skip ";" or "," - empty expressions
-		const char *node_type = ts_node_type(child);
-		if (!strcmp(node_type, "expression_statement")) {
```

Comment:
don't you need to check this anyway?

Reply:
It seems they are used to skip `, `and `;`, maybe using the result of `ts_node_sub_string()` is better. The original way will also skip the statement like `td "stract;"`.

Reply:
@XVilka ?

Reply:
Yes, it was used for any empty expression: https://github.com/rizinorg/tree-sitter-c/blob/bare-types-support/grammar.js#L606

I think this change is OK.